collections/sequence.go

```diff
@@ -32,7 +32,7 @@ func (s Sequence) Peek(ctx context.Context) (uint64, error) {
 	}
 }
 
-// Next returns the next sequence number, and sets the next expected sequence.
+// Next returns the current sequence number, and sets the next expected sequence.
```

Comment:
On the underlying state it's the current sequence number, but it should be the next number you use.

EG: assume you have accounts and you use uint64 as their identifier.

then what `Next()` return is what you should be using as the account ID, not what the current account ID is.

Reply:
Thank you for the clarification! My intention was to provide the same words for the description, because `Sequence.Peek()` are using a word "current".